### ISSUE TITLE
execute mutiple rule 

### ISSUE THREAD

**Issue by M-Razavi** [rorchard/FuzzyJ#3]
Is it possible to execute multiple rule at once?

**Comment by rorchard** [comment]
I don’t believe there is a way to execute multiple rules at once, assuming you mean simultaneously (in parallel).
I may not be clear on exactly what you mean. Normally you have a set of rules that all contribute to 
a solution and the results of those rules are aggregated to determine the solution. This is handled in FuzzyJ
and is the normal way to do things. Again I might be understanding your question fully.

Bob.

**Comment by M-Razavi** [comment]
Let me describe my question with example. In Prolog or Clips you write some rules and facts then run program. The Inference engine works and check all written rules but in FuzzyJ I had to execute each rule separately.

**Comment by rorchard** [comment]
I added the FuzzyJ code to Jess (which is basically a Java implementation of CLIPS). So you can get the benefits of a 
rule based system and use fuzzy reasoning. See http://www.jessrules.com <http://www.jessrules.com/>.

Albert Kong (akong@crisys.com) took over the maintenance of FuzzyJ and FuzzyJess a few years ago after I retired. 
If you have questions you might give him a call. The FuzzyJ documentation describes using FuzzyJess.
If you have problems getting started I can probably help.

Bob.
